src/modules/75-cf/components/SectionToggle/__test__/SectionToggle.test.tsx

```diff
@@ -48,4 +48,25 @@ describe('SectionToggle', () => {
     expect(screen.getByText('cf.shared.segments').parentElement).toHaveClass('active')
     expect(screen.getByText('cf.shared.targets').parentElement).not.toHaveClass('active')
   })
+
+  test('it should render a tooltip when hovering over Targets button', () => {
+    renderComponent()
+    const targetsBtn = screen.getByText('cf.shared.targets')
+    fireEvent.mouseOver(targetsBtn)
+    waitFor(() => {
+      expect(document.getElementsByClassName('Tooltip--tooltipContentWrapper')).toBeInTheDocument()
+      expect(screen.getByRole('link')).toHaveAttribute('href')
+    })
+  })
+
+  test.only('it should render a tooltip with a link when hovering over Target Segments button', () => {
```

Comment:
Looks like you left an `only` in there
```suggestion
  test('it should render a tooltip with a link when hovering over Target Segments button', () => {
```